x/ibc-hooks/contracts/crosschain-swaps/src/msg.rs

```diff
@@ -0,0 +1,89 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Addr, Coin};
+use swaprouter::msg::Slippage;
+
+/// Message type for `instantiate` entry_point
+#[cw_serde]
+pub struct InstantiateMsg {
+    /// This should be an instance of the Osmosis swaprouter contract
+    pub swap_contract: String,
+    /// If set to true, the contract will add a callback request on the packet
+    /// so that it gets notified when an ack is received or if the packet timed
+    /// out. If set to false, any funds sent on a packet that fails after a swap
+    /// will be stuck in this contract.
+    ///
+    /// The information about the packet sender and recovery address is still
+    /// stored, so recovery could be possible after a contract upgrade.
+    pub track_ibc_sends: Option<bool>,
```

Comment:
What are the scenarios that `track_ibc_sends` is not needed?

Reply:
If the chain doesn't have support for sending IBC acks back to the contract (like right now), then there is no way to know if the packet was received or not. So we skip tracking in the contract altogether.

We also remove the memo from the packet (as it would just be forwarded with the packet and have no effect unless the receiving chain handles it)